doc/manual/src/command-ref/nix-collect-garbage.md

```diff
@@ -53,9 +53,10 @@ These options are for deleting old [profiles] prior to deleting unreachable [sto
 
   This is the equivalent of invoking [`nix-env --delete-generations old`](@docroot@/command-ref/nix-env/delete-generations.md#generations-old) on each found profile.
 
-- <span id="opt-delete-older-than">[`--delete-older-than`](#opt-delete-older-than)</span> *period*\
+- <span id="opt-delete-older-than">[`--delete-older-than`](#opt-delete-older-than)</span> *period*|*generations*\
   Delete all generations of profiles older than the specified amount (except for the generations that were active at that point in time).
   *period* is a value such as `30d`, which would mean 30 days.
+  *generations* is a value such as `+3`, which would mean the 3 most recent generations.
```

Comment:
Wouldn't +3 represent the 3 oldest generations?

Reply:
I think it may be clearer to rewrite this as, +N will preserve the N most recent generations and delete anything older.